main.css

```diff
@@ -0,0 +1,500 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+}
+
+.hamb-icon {
+  width: 30px;
+  height: 30px;
+  margin-top: 12px;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+.menu-links {
+  height: 85%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 0 0 10px 10px;
+  background-color: #fff;
+  padding-top: 40px;
+  display: none;
+  flex-direction: column;
+  box-shadow: #ec5242 10px 10px 90px 40px;
+  margin: 0;
+}
+
+.menu-list {
+  margin: 20px 0;
+}
```

Comment:
- [ ] Your header design looks amazing. There is just a small detail that's off, in the original design, this section has its items vertically centered. In order to do this, you could consider adding an `align-self` here. You can read more about align-self in the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self).

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142276481-ac3f8a82-54ea-4cc2-9b57-6f2373dd97c2.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142276285-b0b6a50f-9027-44df-8ee8-59d96f213d0f.png)